src/cljs/rems/administration/components.cljs

```diff
@@ -29,7 +32,12 @@
        (str/join "-")))
 
 (defn- field-validation-message [error label]
-  [:div {:class "invalid-feedback"}
+  ;; XXX: Bootstrap's has "display: none" on .invalid-feedback by default
+  ;;      and overrides that for example when there is a sibling .form-control.is-invalid,
+  ;;      but that doesn't work with checkbox groups, dropdowns, etc., and we anyways
+  ;;      don't need the feature of hiding this div with CSS when it has no content.
+  [:div {:class "invalid-feedback"
+         :style {:display "block"}}
```

Comment:
not a fan of inline styling, I personally think this creates unnecessary "pollution" in the style definition. We can just have ```.invalid-feedback``` class in the styles file

Reply:
Yeah I guess we could move this hack there, especially now that we have two instances of it (see fields.cljs:161)